internal/units/units.go

```diff
@@ -40,6 +42,15 @@ func BytesStringBase2(b int64) string {
 	return toDecimalUnitString(float64(b), 1024.0, base2UnitPrefixes, "B")
 }
 
+func BytesStringBaseEnv(b int64) string {
```

Comment:
`BytesString` is probably sufficient

Reply:
also it might make sense to make it a function pointer and statically initialize it:

```
var BytesString func(int64) string

func init() {
   if ... {
     BytesString = BytesStringBase2
   } else {
     BytesString = BytesStringBase10
   }
}
```

Reply:
Good callouts.

Re: initialization, my impression of Go on the testing front would be that the `units` package would be initialized once, without a chance to set the envvar, and at that point the base-2 case wouldn't be accessible for testing.